doc/modules/manifold.rst

```diff
@@ -502,6 +502,7 @@ three parameters that control the optimization of t-SNE:
 * early exaggeration factor
 * learning rate
 * maximum number of iterations
+* angle
```

Comment:
As is an approximation of the true cost: while the default value should of 0.5 or lower should not impact the results much, setting it to a high value should impact the quality of the embedding.